### PR TITLE
Moved a few messages to messages.conf

### PR DESCRIPTION
**Pull request by EPuncker**
### Pull Request Prelude

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed
Moved a few messages that weren't present in messages.conf yet, please **NOTE** that I'm not a coder in any way, so a review is greatly appreciated as I mostly just copy&pasted pieces from everywhere...

Check specially the rodex.c one!

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

**Comment by 4144** [comment]
see ci, after your changed compilation error

File: src/map/battleground.c [hunk 1]
```diff
@@ -613,7 +613,7 @@ static void bg_match_over(struct bg_arena *arena, bool canceled)
 			bg->queue_pc_cleanup(sd);
 		}
 		if (canceled)
-			clif->messagecolor_self(sd->fd, COLOR_RED, "BG Match Canceled: not enough players");
+			clif->messagecolor_self(sd->fd, COLOR_RED, msg_fd(fd, 83)); // "BG Match Canceled: not enough players."
```
Comment (AnnieRuru) [inline src/map/battleground.c:616]:
```
clif->messagecolor_self(sd->fd, COLOR_RED, msg_fd(sd->fd, 83)); // "BG Match Canceled: not enough players."
```

**Review by AnnieRuru — CHANGES_REQUESTED** [review]
```
battleground.c:616:54: error: 'fd' undeclared (first use in this function); did you mean 'sd'?
  616 |    clif->messagecolor_self(sd->fd, COLOR_RED, msg_fd(fd, 83)); // "BG Match Canceled: not enough players."
      |                                                      ^~
../map/atcommand.h:47:49: note: in definition of macro 'msg_fd'
   47 | #define msg_fd(fd,msg_number) atcommand->msgfd((fd),(msg_number))
      |                                                 ^~
battleground.c:616:54: note: each undeclared identifier is reported only once for each function it appears in
  616 |    clif->messagecolor_self(sd->fd, COLOR_RED, msg_fd(fd, 83)); // "BG Match Canceled: not enough players."
      |                                                      ^~
../map/atcommand.h:47:49: note: in definition of macro 'msg_fd'
   47 | #define msg_fd(fd,msg_number) atcommand->msgfd((fd),(msg_number))
      |                                                 ^~
```
not sure about others, but from travis said only this error

**Review by AnnieRuru — CHANGES_REQUESTED** [review]
```
  475 |   clif->messagecolor_self(sd->fd, COLOR_RED, msg_fd(fd, 477)); // "You cannot create a guild because you are in a clan."
      |                                                     ^~
../map/atcommand.h:47:49: note: in definition of macro 'msg_fd'
   47 | #define msg_fd(fd,msg_number) atcommand->msgfd((fd),(msg_number))
      |                                                 ^~
guild.c:475:53: note: each undeclared identifier is reported only once for each function it appears in
  475 |   clif->messagecolor_self(sd->fd, COLOR_RED, msg_fd(fd, 477)); // "You cannot create a guild because you are in a clan."
      |                                                     ^~
../map/atcommand.h:47:49: note: in definition of macro 'msg_fd'
   47 | #define msg_fd(fd,msg_number) atcommand->msgfd((fd),(msg_number))
      |                                                 ^~
```
please check travis ... and rebase your commit

**Comment by AnnieRuru** [comment]
rebase your commit ....

if you don't know how to do, just force-push your own branch

**Comment by EPuncker** [comment]
> rebase your commit ....
> 
> if you don't know how to do, just force-push your own branch

2020-10-06 15_00_16-MINGW64__c_Users_Iuri_Desktop_Hercules

 no idea how to do it

**Comment by EPuncker** [comment]
Okay I give up about this squash commit things, since any staff of hercules can also git edit my PR, I kindly ask they to do so... or close this PR and I open a new one with everything in one single commit (weird)

**Comment by AnnieRuru** [comment]
1. delete your branch on your computer
2. create a new branch, with the same branch name `msgs`
3. select force push on this branch

**Pull request merged**